src/jv.c

```diff
@@ -571,19 +570,18 @@ static jvp_literal_number* jvp_literal_number_alloc(unsigned literal_length) {
     + sizeof(decNumberUnit) * units
   );
 
+  n->refcnt = JV_REFCNT_INIT;
+  n->num_double = NAN;
+  n->literal_data = NULL;
```

Comment:
What was the reson to move this from new to alloc?

Looks like `jvp_literal_number_new` is the only user of `jvp_literal_number_alloc`, maybe could be merged or keep separate for readability?

Reply:
Also used for negate.

Reply:
Aha sorry 👍